_sass/_components/_logo-bar.scss

```diff
@@ -86,6 +65,8 @@ $logo-bar-scale: 14px;
       text-decoration: none;
       font-size: 1rem;
       justify-content: center;
+      min-width: 25%;
+      transition: margin-left 750ms;
 
       &[href*="tracer"],
       &[href*="dropwizard"] {
```

Comment:
Yeah, you are right. 

The issue is that the initial implementation assumed we would have logos for all tracers/frameworks.  So what I am doing here are small tweaks to make the text and logos align a bit more beautiful. 

A iteration on this implementation would solve these issues for sure.